File: Problem 14.r
# Function to generate Fibonacci series
fibonacci_series <- function(n) {
  fib_ser <- c(0, 1)
  while (length(fib_ser) < n) {
    fib_ser <- c(fib_ser, fib_ser[length(fib_ser)] + fib_ser[length(fib_ser) - 1])
  }
  return(fib_ser)
}

# Generate the Fibonacci series up to 12 terms (as shown in the problem)
n <- 12
fib_seq <- fibonacci_series(n)
cat("First", n, "terms of Fibonacci series:", fib_seq, "\n")

File: Problem 14.py
def fibonacci_series(n):
    fib_series = [0, 1]
    while len(fib_series) < n:
        fib_series.append(fib_series[-1] + fib_series[-2])
    return fib_series

# Generate the Fibonacci series up to 12 terms (as shown in the problem)
n = 12
fib_sequence = fibonacci_series(n)
print(f"First {n} terms of Fibonacci series: {fib_sequence}")
